### ISSUE TITLE
Node "events: all" massively spams event bus in HA

### ISSUE THREAD

**Issue by ShadowRep** [zachowj/node-red-contrib-home-assistant-websocket#1016]
### Describe the bug

I tried to report this here:
[https://github.com/hassio-addons/addon-node-red/issues/1653](https://github.com/hassio-addons/addon-node-red/issues/1653) but this might be more appropriate.

##

I am trying to detect if a HA-timer is canceled at any time.

Using the "events: all" node everything seems to work fine, but after a reboot the event bus in HA
is being spammed massively with messages.
(I can see it in Mosquitto and under: "_HA - Developer Tools - Events - Listen to events_" when I subscribe to "timer.cancelled".)

I can not explain this, since I understand the node should **only** listen and **not generate** messages itself.

Am I doing something completely stupid here or is this a bug? 

## Expected behavior

I expected the node to **fire once**, only **after the timer is canceled**.
Also I expected the node **only to listen** to HA, not send anything itself. (Outside of the Node-Red flow, of course.)

## Actual behavior

It works as expected, when I deploy the flow. After a reboot, the node immediately starts spamming these messages:

```
event_type: timer.cancelled
data:
  entity_id: timer.gastebadfenstertimer
origin: LOCAL
time_fired: "2023-07-29T16:55:30.285594+00:00"
context:
  id: 01H6H9777DMYAFNCC8XJNVBJSP
  parent_id: null
  user_id: null
```

Deactivating the node and deploying stops the spamming.
When I start and cancel the timer again, the spamming also starts again.

## Steps to reproduce

I am using this node:

```
[
    {
        "id": "843a3bb9f85b82b6",
        "type": "server-events",
        "z": "7bfc619dc41e3622",
        "g": "e3d3041bf2d990bd",
        "name": "Timer is canceled",
        "server": "b46bded.64eb82",
        "version": 2,
        "eventType": "timer.cancelled",
        "exposeToHomeAssistant": false,
        "eventData": "{\"entity_id\":\"timer.gastebadfenstertimer\"}",
        "haConfig": [
            {
                "property": "name",
                "value": ""
            },
            {
                "property": "icon",
                "value": ""
            }
        ],
        "waitForRunning": true,
        "outputProperties": [
            {
                "property": "payload",
                "propertyType": "msg",
                "value": "",
                "valueType": "eventData"
            },
            {
                "property": "topic",
                "propertyType": "msg",
                "value": "$outputData(\"eventData\").event_type",
                "valueType": "jsonata"
            }
        ],
        "event_type": "",
        "x": 220,
        "y": 320,
        "wires": [
            [
                "cef8b05c0a81e333",
                "e611c84628c98f9a",
                "af07a9e5eb032778",
                "bd006a328cea24d9"
            ]
        ]
    }
]
```

Import it and deploy the flow. Everything seems fine.
Restart HA. It starts spamming immediately.
Deactivate the node and redeploy and the spamming stops.
After reactivation and redeployment of the node it seems fine,
but after starting and cancelling the timer it starts spamming again.

### To Reproduce

I am using this node:

```
[
    {
        "id": "843a3bb9f85b82b6",
        "type": "server-events",
        "z": "7bfc619dc41e3622",
        "g": "e3d3041bf2d990bd",
        "name": "Timer is canceled",
        "server": "b46bded.64eb82",
        "version": 2,
        "eventType": "timer.cancelled",
        "exposeToHomeAssistant": false,
        "eventData": "{\"entity_id\":\"timer.gastebadfenstertimer\"}",
        "haConfig": [
            {
                "property": "name",
                "value": ""
            },
            {
                "property": "icon",
                "value": ""
            }
        ],
        "waitForRunning": true,
        "outputProperties": [
            {
                "property": "payload",
                "propertyType": "msg",
                "value": "",
                "valueType": "eventData"
            },
            {
                "property": "topic",
                "propertyType": "msg",
                "value": "$outputData(\"eventData\").event_type",
                "valueType": "jsonata"
            }
        ],
        "event_type": "",
        "x": 220,
        "y": 320,
        "wires": [
            [
                "cef8b05c0a81e333",
                "e611c84628c98f9a",
                "af07a9e5eb032778",
                "bd006a328cea24d9"
            ]
        ]
    }
]
```

Import it and deploy the flow. Everything seems fine.
Restart HA. It starts spamming immediately.
Deactivate the node and redeploy and the spamming stops.
After reactivation and redeployment of the node it seems fine,
but after starting and cancelling the timer it starts spamming again.

### Expected behavior

I expected the node to **fire once**, only **after the timer is canceled**.
Also I expected the node **only to listen** to HA, not send anything itself. (Outside of the Node-Red flow, of course.)

### Example Flow

```json
[
    {
        "id": "843a3bb9f85b82b6",
        "type": "server-events",
        "z": "7bfc619dc41e3622",
        "g": "e3d3041bf2d990bd",
        "name": "Timer is canceled",
        "server": "b46bded.64eb82",
        "version": 2,
        "eventType": "timer.cancelled",
        "exposeToHomeAssistant": false,
        "eventData": "{\"entity_id\":\"timer.gastebadfenstertimer\"}",
        "haConfig": [
            {
                "property": "name",
                "value": ""
            },
            {
                "property": "icon",
                "value": ""
            }
        ],
        "waitForRunning": true,
        "outputProperties": [
            {
                "property": "payload",
                "propertyType": "msg",
                "value": "",
                "valueType": "eventData"
            },
            {
                "property": "topic",
                "propertyType": "msg",
                "value": "$outputData(\"eventData\").event_type",
                "valueType": "jsonata"
            }
        ],
        "event_type": "",
        "x": 220,
        "y": 320,
        "wires": [
            [
                "cef8b05c0a81e333",
                "e611c84628c98f9a",
                "af07a9e5eb032778",
                "bd006a328cea24d9"
            ]
        ]
    }
]
```

### This package's version (not the Home Assistant add-on version)

0.55.1

### Is Node-RED running in Docker?

Home Assistant add-on

### Node-RED version

3.0.2

### Node.js version

18.17.0

**Comment by zachowj** [comment]
The events: all node only reports what comes over the event bus of Home Assistant, it doesn't fire any events. Are you sure you don't have anything downstream of the events: all node that is cancelling the timer?

**Comment by ShadowRep** [comment]
I just double checked. I thought I deactivated all nodes after the "events: all" node. But actually I left one active.

In my defense: it was a node that also called the HA service to cancel the (then already cancelled) timer
and _I did not expect that a cancelled timer could be cancelled again._ But it seems that it can be, so a loop was created.

Thanks for challenging me and I am sorry I wasted your time.
